backends-common/redis/src/main/java/org/apache/james/backends/redis/RedisHealthCheck.scala

```diff
@@ -154,3 +156,36 @@ class RedisMasterReplicaHealthCheckPerform(val redisConfiguration: MasterReplica
       .subscribe()
 
 }
+
+class RedisSentinelHealthCheckPerform(val redisURI: RedisURI,
+                                      val maybeSentinelPassword: Option[String],
+                                      val healthcheckTimeout: Duration) extends RedisHealthcheckPerform {
+
+  private val PING_SUCCESS_RESPONSE = "PONG"
+
+  private val redisClient: RedisClient = {
+    maybeSentinelPassword.foreach(password => redisURI.getSentinels.forEach(uri => uri.setCredentialsProvider(RedisCredentialsProvider.from(() => RedisCredentials.just("", password)))))
```

Comment:
This line code (167) looks not relate to the method `RedisClient` 
We can done it (set password credential) for mutable RedisURI at the `SentinelRedisConfiguration.from` 

With that way, we don't need (maybeSentinelPassword: Option[String]) in `SentinelRedisConfiguration` object.
more pretty code